src/sentry/templates/sentry/layout.html

```diff
@@ -49,6 +49,11 @@
   {% asset_url "sentry" "dist/app.js" as asset_url %}
   {% script src=asset_url data-entry="true" %}{% endscript %}
 
+  {% other_js_file as asset_url %}
+  {% if asset_url %}
+    {% script src=asset_url %}{% endscript %}
+  {% endif %}
```

Comment:
@markstory Is this the right way to add an optional script tag? Note this file is coming from the CDN and not Sentry.

Reply:
Any chance you could make it to multiple?

Reply:
@EvanPurkhiser make what to multiple?

Reply:
Like can we have multiple additional assets, that's why I suggested `INJECTED_SCRIPT_ASSETS` (plural), so you could specify a list of files

Reply:
@EvanPurkhiser got it, i'll give it a shot